documentation/src/js_console.md

```diff
@@ -0,0 +1,110 @@
+
+# Forest JavaScript Console
+
+Forest console stands as an alternative to tools like [Curl](https://github.com/curl/curl) or other subcommands found in `forest-cli` for interacting with the Filecoin JSON-RPC API.
+
+## Starting the console
+
+`forest-cli attach` can be used to open a Javascript console connected to your Forest node.
+
+Like for some other `forest-cli` subcommands you will need to pass an admin `--token` given what endpoints you will call.
+
+For a description of different options please refer to the developer documentation [CLI page](https://github.com/ChainSafe/forest/blob/main/documentation/developer_documentation/CLI.md#cli).
+
+## Interactive Use
+
+First start Forest node inside another terminal:
+
+```bash
+forest --chain calibnet --encrypt-keystore false
```

Comment:
Let's not put this `--encrypt-keystore false` in the instructions. It's not required to run the JS console, and it's `true` for a reason.